src/internal/components/dropdown/styles.scss

```diff
@@ -70,6 +70,11 @@
   &.occupy-entire-width {
```

Comment:
Are there any conflicts with this style

Reply:
No, in fact they complement each other. This class is still relevant and makes sure that the dropdown fills the trigger width (when `expandToViewport` is false).